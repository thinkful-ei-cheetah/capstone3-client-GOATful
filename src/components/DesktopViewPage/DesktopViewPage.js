import React from 'react'
import YoutubeSearchResult from '../YoutubeSearchResult/YoutubeSearchResult'
import './DesktopViewPage.css'

export default function DesktopViewPage(props) {
  const renderPreviews = () => {
    return props.videos.map((video, i) => {
      return <YoutubeSearchResult {...video} key={i}/>
    })
  }
  return (
    <div className="desktop-view-page">
      {renderPreviews()}
    </div>
  )
}

DesktopViewPage.defaultProps = {
  videos: []
}