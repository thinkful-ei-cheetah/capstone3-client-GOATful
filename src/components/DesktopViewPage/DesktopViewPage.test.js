import React from 'react';
import ReactDOM from 'react-dom';
import DesktopViewPage from './DesktopViewPage';
import renderer from 'react-test-renderer';


describe('DesktopViewPage Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(< DesktopViewPage />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<DesktopViewPage/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
})
