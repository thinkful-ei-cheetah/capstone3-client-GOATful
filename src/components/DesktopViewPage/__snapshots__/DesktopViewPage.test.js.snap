// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DesktopViewPage Component renders the UI as expected 1`] = `
<div
  className="desktop-view-page"
  id="dvp"
/>
`;
